### DISCUSSION TITLE
Erro in migration:run

### DISCUSSION THREAD

**Discussion by erickximenes** [adonisjs/core#1251] [Help]
In my migration there is a relationship between the tables. However an error happens, the same project I executed a few weeks ago and did not present the error.
![Captura de Tela (21)](https://user-images.githubusercontent.com/67069017/87663532-59fced00-c73a-11ea-843e-3c789644e1fa.png)
![Captura de Tela (22)](https://user-images.githubusercontent.com/67069017/87663538-5b2e1a00-c73a-11ea-9d39-cd9d8cd7d92f.png)
![Captura de Tela (23)](https://user-images.githubusercontent.com/67069017/87663539-5b2e1a00-c73a-11ea-9250-86c96780854f.png)
![Captura de Tela (24)](https://user-images.githubusercontent.com/67069017/87663541-5bc6b080-c73a-11ea-9767-be3598ee84b8.png)
![Captura de Tela (25)](https://user-images.githubusercontent.com/67069017/87663542-5c5f4700-c73a-11ea-8d93-74f75908e97b.png)
![Captura de Tela (26)](https://user-images.githubusercontent.com/67069017/87663544-5c5f4700-c73a-11ea-8379-ee6709630601.png)
![Captura de Tela (27)](https://user-images.githubusercontent.com/67069017/87663545-5cf7dd80-c73a-11ea-8fa8-42e5eaf01364.png)

**Adonis version: 4.0.12
Node version: 12.18.1**

**Comment by fnoquiq** [comment]
You tried to run `adonis migration:refresh`?

**Reply by erickximenes** [reply]
Yes

**Comment by thetutlage** [comment]
I'll appreciate, if you share the code directly vs pasting dozens of screenshots. It's hard to follow them.

Regarding the FK, you have to mark the column `unsigned`.

```js
table.integer('user_id').unsigned()
```

**Comment by erickximenes** [comment]
https://drive.google.com/file/d/1y1vXB35yL_YTOZxQ-y7LtTbukbGCZkeb/view?usp=sharing

**Reply by thetutlage** [reply]
Any issues with sharing the code here? 

- One time you dump a bunch of screenshots.
- After politely asking, now you are sharing link to google drive?

No one is getting paid to out of their way to help you.

**Comment by thetutlage** [comment]
Closing discussion is low quality
